doc/experiment.rst

```diff
@@ -1,327 +1,423 @@
+gives you access to functions associated with the :ref:`Orchestrator API<orc_api>`.
+
+Colocated Orchestrator
+^^^^^^^^^^^^^^^^^^^^^^
+An ``Orchestrator`` can be created to share the compute node(s)
+and resources with a SmartSim ``Model``. In this case, the database
+is deployed on the same compute hosts as a Model instance
+defined by the user. In this deployment, the database is not connected
+together in a cluster and each shard of the database is addressed
+individually by the processes running on that compute host.
+If the SmartSim ``Model`` spans more than one
+compute node, the colocated database will also span all of the
+compute nodes. The colocated deployment strategy for the Orchestrator
+is ideal for use cases where a SmartSim ``Model`` is run on a compute node
+that has hardware accelerators (e.g. GPUs) and low-latency inference is
+a critical component of the workflow.
+
+Colocated Deployment Diagram
+""""""""""""""""""""""""""""
+During colocated deployment, a SmartSim orchestrator (the database) runs on the same
+compute node(s) as a Smartsim model (the application).
+This type of deployment is optimal for high data inference scenarios.
+
+Below is an image illustrating communication
+between a colocated model spanning multiple compute nodes, and the database
+running on each application compute node. A single SmartRedis client is initialized
+for the colocated Orchestrator and is used to communicate with the application.
+Data is streamed from the application to the database via the client on the same node.
+
+.. figure:: images/co-located-orc-diagram.png
+
+Initialize a Colocated Orchestrator
+"""""""""""""""""""""""""""""""""""
+To create an ``Orchestrator`` that shares compute resources with a ``Model``
+SmartSim entity, use the ``model.colocate_db()`` factory method.
+In this case, the database
+is created via the SmartSim Model API function ``model.colocate_db``.
+The :ref:`Model API<model_api>` is accessed once a ``Model`` object has been initialized.
+
+
+Multi-db support
+^^^^^^^^^^^^^^^^
+SmartSim supports multi-database functionality, enabling an experiment
+to have several concurrently launched ``Orchestrator(s)``. If there is
+a need to launch more than one ``Orchestrator``, the ``Experiment.create_database()``
+function mandates the specification of a unique database identifier,
+denoted by the `db_identifier` argument, per created orchestrator.
+
+The `db-identifier` is used to reference SmartSim
+orchestrator(s) from application client code. This is particularly
+useful in instances where an orchestrator is colocated with a SmartSim
+model for low-latency inference and another Orchestrator is launched to
+handle other aspects of the workflow such as visualization and ML model
+training. More detailed information on the ideal use cases for orchestrator(s)
+and co-located ``Orchestrator(s)`` are available in sections... (update this when use cases added)
```

Comment:
this will be updated when Orch section is merged

Reply:
Following up on this.  Do we know the name of the Orchestrator section?  Can we put in a dead link instead of this text and add a TODO on the orchestrator ticket to verify this link?